tests/test_zeus_client.py

```diff
@@ -68,23 +81,22 @@ def test_validate_dates(self):
     @mock.patch('zeus.interfaces.rest.requests')
     def test_post_empty_log(self, mock_requests):
         logs = []
+        url = '{}/logs/{}/ZeusTest'.format(FAKE_SERVER, FAKE_TOKEN)
```

Comment:
I'm not planning to refactor using `urljoin`. because 

```python
urlparse.urljoin('/abc/', 'def')
# => /abc/def

urlparse.urljoin('/abc', 'def')
# => '/def'

urlparse.urljoin('abc', 'def')
# => 'def'

urlparse.urljoin('abc', '/def')
# => '/def'

urlparse.urljoin('/abc', '/def')
# => '/def'

urlparse.urljoin('/abc/', '/def')
# => '/def'
```

So

```python
# From
url = '{}/logs/{}/ZeusTest'.format(FAKE_SERVER, FAKE_TOKEN)

# To
url = urlparse.urljoin(FAKE_SERVER, '/logs/')
url = urlparse.urljoin(url, FAKE_TOKEN + '/')
url = urlparse.urljoin(url, ZeusTest)
```

looks bit redundant... (or am i missing something...?)